data/governments.txt

```diff
@@ -1200,12 +1211,15 @@ government "People's Houses (Hostile)"
 		Author -0.01
 		Successor -0.01
 		"People's Houses" 0.01
+	bribe 0.05
+	fine 0
 	"friendly hail" "people's houses friendly hail"
 	"friendly disabled hail" "successor friendly disabled hail"
 	"hostile hail" "people's houses hostile hail"
 	"hostile disabled hail" "high houses hostile disabled hail"
-	bribe 0.05
-	fine 0
+	language "Successor"
+	raid "Large Old Houses" 2 6
+	raid "Large New Houses" 6 0
```

Comment:
Thoughts on this and the line above it? Currently, if there is a _lower_ attraction, there will be an Old Houses fleet attacking. The higher the attraction, the more likely it will be a New Houses.
Should it be vice versa, so the Old Houses aren't tempted as much by small cargo?

Reply:
I like the idea, but doesn't `raid` only apply based on the _player's_ cargo/deterrence stats? [The wiki](https://github.com/endless-sky/endless-sky/wiki/CreatingGovernments#raid) also notes that "In order to be spawned, the named fleet's government must also be hostile to the player," which suggests to me that this wouldn't do anything regardless.

Reply:
It won't yet, true, so we don't have to add it, but it might be best to in the initial PR.

Reply:
I think it's probably best to keep it out. If other sorts of raid behavior are eventually implemented, I think it would preferable to handle this at that time when it's clear how to do it without potentially introducing unintended behavior.